packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -70,13 +70,27 @@
         <protected>false</protected>
         <shortDescription>Clear filter</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ClearFilterFacet</fullName>
+        <value>Clear filter for the {{0}} facet.</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Clear filter for the potato facet.</shortDescription>
```

Comment:
```suggestion
        <shortDescription>Clear filter for the filetype facet.</shortDescription>
```

Let's keep the examples cleaner a little bit ;)

Reply:
filetype-->file type